codepropertygraph/src/main/scala/io/shiftleft/codepropertygraph/cpgloading/CpgOverlayLoader.scala

```diff
@@ -20,9 +22,17 @@ private[cpgloading] object CpgOverlayLoader {
     */
   def load(filename: String, baseCpg: Cpg): Unit = {
     val applier = new CpgOverlayApplier(baseCpg.graph)
-    ProtoCpgLoader.loadOverlays(filename).foreach { overlay =>
-      applier.applyDiff(overlay)
-    }
+    ProtoCpgLoader.loadOverlays(filename)
+      .map { overlays: Iterator[CpgOverlay] =>
+        overlays.foreach(applier.applyDiff)
+      }
+      .tried
+      .recover {
+        case e: IOException =>
+          e.printStackTrace()
```

Comment:
please use a logger. this also runs in `mesh` (part of the CS repo), where this exception would get lost